kie-api/src/main/java/org/kie/api/definition/process/Node.java

```diff
@@ -87,4 +87,10 @@
      */
     Map<String, Object> getMetaData();
 
+    /**
+     * Type of this node
+     * @return possible type of nodes
+     */
+    NodeType getNodeType();
```

Comment:
Shouldn't it be defined as "default" to avoid breaking backward compatibility?

Reply:
+1

Reply:
hmmm I prefer to implement it in all possible places, since the only implementors are suppose to be us. In any case I will run a FDB

Reply:
I agree with QE. set default and DEFAULT or INTERNAL, GENERIC as return type

Reply:
ok, setted to default, INTERNAL as default value